CHANGELOG.md

```diff
@@ -2,24 +2,35 @@
 
 ## Unreleased
```

Comment:
This is missing square brackets, and please move the link up to be on the next line.